composer.json

```diff
@@ -15,9 +15,9 @@
     "require": {
         "php": "^7.0",
         "matthiasnoback/symfony-config-test": "^3.0",
-        "symfony/dependency-injection": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0",
-        "symfony/yaml": "^2.7|^3.0"
+        "symfony/dependency-injection": "^2.7|^3.0|^4.0",
```

Comment:
Conditionals should be done with `||` instead of single `|`, see: https://getcomposer.org/doc/articles/versions.md#version-range